networkx/algorithms/isomorphism/isomorph.py

```diff
@@ -108,10 +108,8 @@ def faster_could_be_isomorphic(G1,G2):
     if G1.order() != G2.order(): return False
 
     # Check local properties
-    d1=list(G1.degree().values())
-    d1.sort()
-    d2=list(G2.degree().values())
-    d2.sort()
+    d1=sorted(dict(G1.degree()).values())
```

Comment:
```d1 = sorted( G1.degree(), key=itemgetter(1)) # sort by degree```
(see rcm.py at bottom of page for itemgetter...You'll have to import that from the Python operator module)